apps/prairielearn/src/models/enrollment.ts

```diff
@@ -118,3 +141,61 @@ export async function generateAndEnrollUsers({
     return users;
   });
 }
+
+export async function selectEnrollmentById({ id }: { id: string }) {
```

Comment:
not sure if we need a pattern for marking if a model returns a safe vs non safe enrollment. E.g. we would never want to accidentally insert a safe enrollment into audit events.

Reply:
It looks like this was resolved.